data/json/monstergroups/blob.json

```diff
@@ -19,7 +19,8 @@
       { "monster": "mon_blob_small", "weight": 200, "cost_multiplier": 0 },
       { "monster": "mon_blob_small", "weight": 100, "cost_multiplier": 0, "pack_size": [ 2, 12 ] },
       { "monster": "mon_blob", "weight": 695, "cost_multiplier": 3, "pack_size": [ 5, 10 ] },
-      { "monster": "mon_blob_large", "weight": 5, "cost_multiplier": 6, "pack_size": [ 5, 10 ] }
+      { "monster": "mon_blob_large", "weight": 5, "cost_multiplier": 6, "pack_size": [ 5, 10 ] },
+      { "monster": "mon_aquatic_blob", "weight": 200, "cost_multiplier": 0, "pack_size": [ 5, 10 ] }
```

Comment:
I don't see anything here to guarantee that they will spawn on water. As far as I can tell they'll just spawn among other slimes, so in slimes pits and labs pretty far from any pond.